my-react-app/src/app/screens/Game/index.js

```diff
@@ -1,52 +1,24 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { actions } from '../../../redux/game/actions';
 
 import Board from './components/Board';
 import styles from './styles.scss';
 import calculateWinner from './utils.js';
 
 class Game extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      history: [
-        {
-          squares: Array(9).fill(null)
-        }
-      ],
-      stepNumber: 0,
-      xIsNext: true
-    };
-  }
-
   handleClick(i) {
-    const history = this.state.history.slice(0, this.state.stepNumber + 1);
-    const current = history[history.length - 1];
-    const squares = current.squares.slice();
-    if (calculateWinner(squares) || squares[i]) {
-      return;
-    }
-    squares[i] = this.state.xIsNext ? 'X' : 'O';
-    this.setState({
-      history: history.concat([
-        {
-          squares
-        }
-      ]),
-      stepNumber: history.length,
-      xIsNext: !this.state.xIsNext
-    });
+    this.props.dispatch({ type: actions.handleClick, pos: i });
```

Comment:
Move this to a `mapDispatchToProps` function. Also, the actions should have a `type` and `payload` properties.